app/src/utils/comparisonModels.ts

```diff
@@ -14,8 +14,9 @@ export const isComparisonModelName = (modelName: string) =>
 
 export const COMPARISON_MODEL_NAMES: Record<ComparisonModel, string> = {
   GPT_3_5_TURBO: "gpt-3.5-turbo-1106",
-  GPT_4_1106_PREVIEW: "gpt-4-1106-preview",
+  GPT_4_0125_PREVIEW: "gpt-4-0125-preview",
```

Comment:
We rely on values in some places because it is compatible with openAi `model` field. I renamed it by adding an object as a value, that has an OpenAI compatible `name` and a human-readable `slug`.